### ISSUE TITLE
Why timestamps are disabled?

### ISSUE THREAD

**Issue by adriandrozdz** [rtconner/laravel-tagging#156]
I would like to implement a feature where I retreive the most popular tags over the last month and an obvious solution would be to get them by timestamp which is disabled here by default and hardcoded in Tagged model. 

Is there any way I can easily enabled it? Writing only migrations to add timestamps won't work as the property is set to false.

```public $timestamps = false;```

**Comment by rtconner** [comment]
Yes, that is the preferred solution. Most implementations do not need timestamps.
